app/views/handlers/thank_you.py

```diff
@@ -36,12 +32,8 @@ def __init__(
         self._schema: QuestionnaireSchema = schema
         self._submitted_at = submitted_at
 
-        self._is_census_theme = cookie_session.get("theme") == "census"
-        self.template = (
-            self.CENSUS_THANK_YOU_TEMPLATE
-            if self._is_census_theme
-            else self.DEFAULT_THANK_YOU_TEMPLATE
-        )
+        # self._is_census_theme = cookie_session.get("theme") == "census"
```

Comment:
Comment left behind

Reply:
Some of these comments will be removed in the Email Confirmation PR and then it will removed in here when that PR merges.